pom.xml

```diff
@@ -141,6 +141,9 @@
                 <configuration>
                     <java>
                         <removeUnusedImports />
+                        <importOrder>
+                            <order>java,javax,jakarta,org,com</order>
+                         </importOrder>
                         <eclipse>
                             <file>eclipse-codeStyle.xml</file>
                         </eclipse>
```

Comment:
This will need change to add 
```
  -                            <file>eclipse-codeStyle.xml</file>
  +                            <file>java-formatter.xml</file>
```
and also 
```
  +                <dependencies>
  +                    <dependency>
  +                        <groupId>org.jboss.pnc</groupId>
  +                        <artifactId>ide-config</artifactId>
  +                        <version>1.0.0</version>
  +                    </dependency>
  +                </dependencies>
```
(well, the version needs to be released first tbh)

Reply:
Also the `eclipse-codeStyle.xml`  in the top level can be deleted.